framework/src/wifi_manager/wifi_manager.c

```diff
@@ -1479,6 +1495,27 @@ wifi_manager_result_e wifi_manager_remove_config(void)
 	return WIFI_MANAGER_SUCCESS;
 }
 
+
+wifi_manager_result_e wifi_manager_get_stats(wifi_manager_stats_s *stats)
+{
+	wifi_manager_result_e wret = WIFI_MANAGER_INVALID_ARGS;
+	if (stats) {
+		stats->connect = g_manager_info.stats[CB_STA_CONNECTED];
+		stats->connectfail = g_manager_info.stats[CB_STA_CONNECT_FAILED];
+		stats->disconnect = g_manager_info.stats[CB_STA_DISCONNECTED];
+		stats->reconnect = g_manager_info.stats[CB_STA_RECONNECTED];
+		stats->joined = g_manager_info.stats[CB_STA_JOINED];
+		stats->left = g_manager_info.stats[CB_STA_LEFT];
+		stats->scan = g_manager_info.stats[CB_SCAN_DONE];
+		/* SoftAP mode has no callback */
+		stats->softap = g_manager_info.stats[CB_MAX];
```

Comment:
Instead of CB_MAX, CB_SOFT_AP might be better for readibility.

Reply:
I agree with you. CB_SOFTAP_DONE is included and CB_MAX denotes the maximum number of CB functions. Thanks.